packages/extension/src/store/PAStore.ts

```diff
@@ -35,6 +41,24 @@ class PAStore extends DataStore {
       [uniqueAuctionId: string]: singleAuctionEvent[];
     };
   } = {};
+
+  /**
+   * The auction event of the tabs (Interest group access as well as interest group auction events).
+   */
+  prebidEvents: {
```

Comment:
`PAStore` store is growing larger. Prebid is a totally different auction than PA, why don't we keep them separate or extend class.

Reply:
Related [comment](https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/1078#discussion_r2100473668)